reference/7.3/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -14,66 +14,168 @@ Explains language modes and their effect on PowerShell sessions.
+  referenced in restricted language mode or a Data section is being
+  referenced.
+
+When you run the `$ExecutionContext.SessionState.LanguageMode` command in a
+NoLanguage session, PowerShell returns the **ScriptsNotAllowed** error message.
+
+- **ScriptsNotAllowed**: The syntax isn't supported by this runspace. This
+  might be because it's in no-language mode.
+
+## Finding the language mode of a session configuration
+
+When a session configuration is created using a session configuration file, the
+session configuration has a **LanguageMode** property. You can find the
+language mode by getting the value of the **LanguageMode** property.
+
+```powershell
+(Get-PSSessionConfiguration -Name Test).LanguageMode
+FullLanguage
+```
+
+On other session configurations, you can find the language mode indirectly by
+finding the language mode of a session that's created using the session
 configuration.
 
-All PowerShell sessions have a language mode, including PSSessions that you
-create by using the `New-PSSession` cmdlet, temporary sessions that use the
-ComputerName parameter, and the default sessions that appear when you start
-PowerShell.
+## Setting the language mode
 
-Remote sessions are created by using the session configurations on the remote
-computer. The language mode set in the session configuration determines the
-language mode of the session. To specify the session configuration of a
-PSSession, use the ConfigurationName parameter of cmdlets that create a
-session.
+The language mode in a PowerShell session can be set through the built in
+`$ExecutionContext` variable.
+
+```powershell
+$ExecutionContext.SessionState.LanguageMode = "ConstrainedLanguage"
+```
+
+However, doing this is only useful for experimenting with language modes.
+Language modes are intended to provide added security to PowerShell sessions
+for specific contexts.
+
+Languages modes are set when you use a system application control policy or
+create a session configuration.
+
+### Using a system application control policy
+
+PowerShell automatically runs in `ConstrainedLanguage` mode when it's running
+under a system application control policy. The application control
+policies detected are AppLocker and Windows Defender Application Control (WDAC)
+on Windows platforms.
+
+PowerShell applies other restrictions besides language modes when it detects an
+application control policy. For example, there are additional restrictions to
+dot-sourcing and module importing under a policy.
+
+When a PowerShell session is started under a policy, it runs in
+`ConstrainedLanguage` mode. Beginning in PowerShell 7.4, the start up banner
+display includes a message indicating it's running in that mode. This allows
+users to have a usable interactive shell experience, running cmdlets and native
+commands, as well as access to basic language elements. But the user can't
+access PowerShell, .NET, or COM APIs that could be abused by a malicious actor.
```

Comment:
Nit: Should the note about 7.4 be included in this version?

Suggesting a reword of the second part of the paragraph:
```suggestion
When a PowerShell session is started under a policy, it runs in
`ConstrainedLanguage` mode. Beginning in PowerShell 7.4, the start up banner
display includes a message indicating it's running in that mode. This mode
allows for a usable interactive shell experience while limiting access to
features and APIs that could be abused by a malicious actor. Users can run
cmdlets and native commands and have access to basic language elements. Access
to PowerShell, .NET, and COM APIs is restricted.
```
Something about the construction was a little difficult for me to parse. The intent of this rewrite is to consolidate the message (usable interactive shell, limited functionality) and then clarify the availability.